operations-stdlib/src/commonMain/kotlin/string/Trim.kt

```diff
@@ -0,0 +1,61 @@
+package string
+
+import operation.StandardLogicOperation
+import utils.asList
+import utils.secondOrNull
+import utils.thirdOrNull
+
+object Trim : StandardLogicOperation, StringUnwrapStrategy {
+    override fun evaluateLogic(expression: Any?, data: Any?): Any?  {
+        return when (val firstItem = expression.asList.firstOrNull()) {
+            is String -> firstItem.trimElements(' ', TrimMode.BothEnds)
```

Comment:
We discussed it on slack that we dont want to use defaults in the library. When some arguments are missing we should treat it as a wrong input and usually return null. What we decided is to declare operation defaults on backend/dsl library.

Reply:
👍